jobs/build.Jenkinsfile

```diff
@@ -319,18 +314,18 @@ lock(resource: "build-${params.STREAM}") {
         // reserving our build ID before we fork off multi-arch builds.
         stage('Archive OSTree') {
             if (uploading) {
-              // run with --force here in case the previous run of the
-              // pipeline died in between buildupload and bump_builds_json()
-              shwrap("""
-              export AWS_CONFIG_FILE=\${AWS_BUILD_UPLOAD_CONFIG}
-              cosa buildupload --force --skip-builds-json --artifact=ostree \
-                  s3 --acl=public-read ${s3_stream_dir}/builds
-              """)
-              pipeutils.bump_builds_json(
-                  params.STREAM,
-                  newBuildID,
-                  basearch,
-                  s3_stream_dir)
+                // run with --force here in case the previous run of the
+                // pipeline died in between buildupload and bump_builds_json()
+                pipeutils.shwrapWithAWSBuildUploadCredentials("""
+                cosa buildupload --force --skip-builds-json --artifact=ostree \
+                    s3 --aws-config-file=\${AWS_BUILD_UPLOAD_CONFIG} \
```

Comment:
Do we actually need `--aws-config-file` since `withAWSBuildUploadCredentials` sets ` AWS_CONFIG_FILE` as we did directly before here?

Similarly in some other hunks.

Reply:
We do if we want it to work in a remote session.